.cargo/audit.toml

```diff
@@ -0,0 +1,25 @@
+
+
+[advisories]
+# 2024-0388 derivative is unmaintained a PR is open on log4rs but blocked...
+# 2018-0015 is on clipps use of term
```

Comment:
It's a package called term. Which us unmaintained. But given that pat pointed out I don't need to depend on clippy since it's part of rustup I can drop this.